flaml/automl.py

```diff
@@ -2326,20 +2326,20 @@ def _search_parallel(self):
                 wall_time = result.get("wall_clock_time")
                 if wall_time is not None:
                     self._state.time_from_start = wall_time
+                self._iter_per_learner[estimator] += 1
```

Comment:
Do we also need to change line 2561 (the sequential case)?

Reply:
line 571 is needed for search_sequential()

Reply:
In the sequential case, we use this field in line 2852, so we can't change line 2561. Maybe I should use a different variable to avoid the inconsistency.